node/cmd/ccq/http.go

```diff
@@ -38,16 +42,24 @@ func (s *httpServer) handleQuery(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// The API key should be in the header.
+	apiKey, exists := r.Header["X-Api-Key"]
+	if !exists || len(apiKey) != 1 {
```

Comment:
The length of the api key is exactly 1?

Reply:
That map look up returns a slice, so it is checking to make sure that there is only API key in the header. I updated the comment.